statsmodels/genmod/cov_struct.py

```diff
@@ -583,11 +582,10 @@ def update_nogrid(self, params):
         dep_params /= dn
         self.dep_params = dep_params[1:] / dep_params[0]
 
-
     def covariance_matrix(self, endog_expval, index):
 
         if self.grid:
-            return self.covariance_matrix_grid(endog_expal, index)
+            return self.covariance_matrix_grid(endog_expval, index)
```

Comment:
Yes, flake8 picked this up.  We don't use `covariance_matrix` internally anymore, instead we do everything through `covariance_matrix_solve`, which has decent test coverage.  But `covariance_matrix` is useful for people who want to see the actual fitted covariance matrices.  I will add some tests.